extension-api/app/main_simple.py

```
@@ -0,0 +1,253 @@
+
+    # For development, accept any API key that starts with "dev-"
+    if api_key.startswith("dev-"):
+        return {"status": "authenticated", "api_key": api_key[:8] + "..."}
+
+    raise HTTPException(status_code=401, detail="Invalid API key")
+
+
+# Export VS Code extensions endpoint
+@app.post("/api/export-collection", response_model=ExportCollectionResponse)
+async def export_vscode_extensions(
+    request: Request,
```

Ruff (0.14.0)

Unused function argument: `request`

(ARG001)